programs/bubblegum/src/lib.rs

```diff
@@ -64,31 +65,40 @@ pub struct Mint<'info> {
     /// CHECK: This account is neither written to nor read from.
     pub mint_authority: Signer<'info>,
     #[account(
-        seeds = [merkle_slab.key().as_ref()],
-        bump,
+    seeds = [merkle_slab.key().as_ref()],
+    bump,
     )]
     /// CHECK: This account is neither written to nor read from.
     pub authority: UncheckedAccount<'info>,
     #[account(
-        mut,
-        seeds = [NONCE_PREFIX.as_ref()],
-        bump,
+    mut,
+    seeds = [NONCE_PREFIX.as_ref(), merkle_slab.key().as_ref()],
+    bump,
     )]
     pub nonce: Account<'info, Nonce>,
```

Comment:
given than we have 1 nonce per tree now, we should definitely consolidate the nonce and authority into the same account -> 2x max tree capacity